app/helpers/kaui/locale_helper.rb

```diff
@@ -0,0 +1,20 @@
+module Kaui
+  module LocaleHelper
+
+    def get_available_locales
+      locale_yaml_file = File.join(File.dirname(__FILE__), 'locale_helper.yml')
```

Comment:
Did you check this works in the killbill-admin-ui-standalone war (`File` operations don't always work there)?

Reply:
Yes, I tested it with an executable war of the killbill-admin-ui-standalone and it work without problem.